client/log.go

```diff
@@ -135,3 +157,8 @@ func (c *Log) SetHTTPRequestSection(section string) Client {
 func (c *Log) ResetHTTPRequestSection() Client {
 	return c.SetHTTPRequestSection(bucket.SectionRequest)
 }
+
+// Handler returns metrics endpoint for prometheus backend
+func (c *Log) Handler() http.Handler {
+	return nil
```

Comment:
we'd better return handler always - in this case handler that, for example, returns `405` status code

Reply:
still valid =)